cookbooks/2d_annulus_visualization/doc/2d_annulus_visualization.md

```diff
@@ -0,0 +1,117 @@
+(sec:cookbooks:2d_annulus_visualization)=
+# Two-dimensional cylindrical annulus visualization
+
+*This section was contributed by Madeleine Kerr.*
+
+This cookbook describes how to use the Python package pyvista to read in time series data from ASPECT .pvd and .pvtu output files (commonly written as 'solution.pvd' or 'solution-00000.pvtu' within the automatically generated [output/solution/] folder). This particular script has the built-in assumption that the model has a 2D cylindrical annulus geometry since it extracts radial and angular velocity from x,y coordinates and velocities and uses the inner and outer radius to threshold the mesh and perform operations.
+
+The input file to generate the model that created the accompanying figures can be found at [cookbooks/twoD_annulus_visualization/2D_annulus_example.prm]. The ASPECT parameter file is a simple modification of the onset_of_convection cookbook contributed by Juliane Dannberg.
+
+## How to get started
+Since this Python script utilizes pyvista, you should install this package using anaconda following the instructions in the README.md file found in the directory [/aspect/contrib/python/]. Other python scripts that contribute to ASPECT's  functionality can be found within that directory, as well as another 2D pyvista visualization tutorial for a box geometry called 'pyvista_2d_example.py', contributed by John Naliboff.
```

Comment:
```suggestion
Since this Python script utilizes pyvista, you should install this package using anaconda following the instructions in the README.md file found in the directory [/aspect/contrib/python/]. Other python scripts that contribute to ASPECT's  functionality can be found within that directory, as well as another 2D pyvista visualization tutorial for a box geometry called 'pyvista_2d_example.py'.
```